src/Components/ThemeMode/ThemeMode.tsx

```diff
@@ -0,0 +1,9 @@
+type ThemeModeProps = {
+  message : string
+}
+
+const ThemeMode = ({message}: ThemeModeProps) => {
+  return <h1>{message}</h1>;
```

Comment:
you can make an implicit return if there is no logic inside 
` const ThemeMode = ({message}: ThemeModeProps) => <h1>{message}</h1>`

Reply:
but what this theme mode actually does? 
H1 tags are used on page mostly as primary title, somthing that should define type of the content of whole page, key word for SEO robots searches (indexing pages for google in example)

Reply:
theme mode component is now deleted - it will be added in the future